### PR TITLE
add some modification to support cpu well

### PR DESCRIPTION
**Pull request by Guo-Peilin**

### CODE REVIEW

File: torchbenchmark/util/backends/torchdynamo.py [hunk 1]
```diff
@@ -21,14 +21,19 @@ def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dy
     return args, extra_args
 
 def apply_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', args: argparse.Namespace, precision: str):
-    torchdynamo.config.raise_on_backend_error = False
+    torchdynamo.config.raise_on_backend_error = True
```
Comment (zzpmiracle) [inline torchbenchmark/util/backends/torchdynamo.py]: Not raise this error in CI, just for debug

File: torchbenchmark/util/backends/torchdynamo.py [hunk 2]
```diff
@@ -21,14 +21,19 @@ def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dy
...
 
     if args.torchdynamo == "fx2trt" and precision == "fp16":
         dynamo_optimizer = torchdynamo.optimize(torchdynamo.optimizations.backends.fx2trt_compiler_fp16)
     elif "blade" in args.torchdynamo:
         dynamo_optimizer = torchdynamo.optimize(functools.partial(blade_optimize_dynamo, enable_fp16=precision=="fp16", use_trt=args.trt))
+    elif "ipex" in args.torchdynamo:
+        if precision == "fp32":
+            dynamo_optimizer = torchdynamo.optimize(torchdynamo.optimizations.backends.ipex_fp32)
+        else:
+            dynamo_optimizer = torchdynamo.optimize(torchdynamo.optimizations.backends.ipex_bf16)
```
Comment (zzpmiracle) [inline torchbenchmark/util/backends/torchdynamo.py]: torchbench not support bf16 now.

File: torchbenchmark/util/backends/torchdynamo.py (hunk 1 above)
Comment (Guo-Peilin) [reply torchbenchmark/util/backends/torchdynamo.py]: fixed

File: torchbenchmark/util/backends/torchdynamo.py (hunk 2 above)
Comment (Guo-Peilin) [reply torchbenchmark/util/backends/torchdynamo.py]: fixed

**Review by zzpmiracle — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**